Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/ActorData.xml

```diff
@@ -1062,7 +1062,7 @@
         <SoundArray index="Yes" value="NydusWorm_What"/>
         <SoundArray index="Construction" value="Zerg_BuildingMorphWhat"/>
         <UnitFlags index="ShowBlobShadow" value="0"/>
-        <UnitIcon value="AP\Assets\Textures\OmegaNydusDDSIcon.dds"/>
+        <UnitIcon value="Assets\Textures\talent-kerrigan-level08-greaternydusworm.dds"/>

```

Comment:
The old icon path also shows up in 4 places in ButtonData.xml

Reply:
Thank you, those have been changed as well. thought it was just the actor data with that information

Reply:
The other icons you added, ArmyPanelIcon and MinimapIcon, probably also exist in the game files, I just don't have their exact filenames on-hand.